graylog2-server/src/main/java/org/graylog2/indexer/searches/Searches.java

```diff
@@ -263,11 +263,19 @@ public SearchResult search(SearchesConfig config) {
         return new SearchResult(r.getHits(), indices, config.query(), request.source(), r.getTook());
     }
 
-    public TermsResult terms(String field, int size, String query, String filter, TimeRange range) {
+    public TermsResult terms(String field, int size, String query, String filter, TimeRange range, Sorting.Direction sorting) {
+        Terms.Order termsOrder;
+
         if (size == 0) {
             size = 50;
         }
 
+        if (sorting == Sorting.Direction.DESC){
+            termsOrder = Terms.Order.count(false);
```

Comment:
Why not [`Terms.Order#term(boolean)`](https://static.javadoc.io/org.elasticsearch/elasticsearch/2.4.4/org/elasticsearch/search/aggregations/bucket/terms/Terms.Order.html#term(boolean)) here?